ui/packages/catalog-ui-search/src/main/webapp/react-component/results-export/container.tsx

```diff
@@ -82,23 +82,32 @@ class ResultsExport extends React.Component<Props, State> {
     fetch(`./internal/transformers/${this.getTransformerType()}`)
       .then(response => response.json())
       .then((exportFormats: ExportFormat[]) => {
-        const resultsAttributes = Array.from(
-          new Set(
-            this.props.results
-              .map((result: Result) => result.attributes)
-              .reduce((result, arr) => result.concat(arr))
-          )
-        )
+        const resultsAttributes: any = {}
+        this.props.results.map((result: Result) => {
+          result.attributes.forEach((attr: any) => {
+            if (resultsAttributes[attr] === undefined) {
```

Comment:
you could do something like
```
resultsAttributes[attr] = resultsAttributes[attr] || []
resultsAttributes[attr].push(result.id)
```

 and then populate it afterwards. maybe there's a better one-liner way to do it